scripts/cm_test_all_sandia

```diff
@@ -352,6 +358,12 @@ do
     --with-layouts*)
       KOKKOSKERNELS_LAYOUTS="${key#*=}"
       ;;
+    --no-default-eti*)
+      NO_DEFAULT_ETI=True
+      ;;
+    --with-layouts*)
```

Comment:
@e10harvey I think this should be changed, --with-layouts option is a few lines above
```suggestion
    --with-spaces*)
```